Source/RapyutaSimulationPlugins/Private/Drives/RobotVehicleMovementComponent.cpp

```diff
@@ -23,7 +23,7 @@ void URobotVehicleMovementComponent::UpdateMovement(float DeltaTime)
     const FQuat OldRotation = UpdatedComponent->GetComponentQuat();
 
     FVector position = UpdatedComponent->ComponentVelocity * DeltaTime;
-    FQuat DeltaRotation(FVector::ZAxisVector, AngularVelocity.Z * DeltaTime);
+    FQuat DeltaRotation(FVector::ZAxisVector, InversionFactor * AngularVelocity.Z * DeltaTime);

```

Comment:
@yuokamoto not sure why AMRAIVehicle moves in a different style from TurtleBotBurger tho?
@praphulkallakuri And thus, based on the rationale, we might need a more specific name for `InversionFactor` I suppose.

Reply:
@yuokamoto not sure why AMRVehicle moves in a [different](https://github.com/rapyuta-robotics/io_amr_UE/pull/28#discussion_r736254866) style from TurtleBotBurger tho?
@praphulkallakuri And thus, based on the rationale, we might need a more specific name for `InversionFactor` I suppose.